server/render.js

```diff
@@ -21,6 +21,21 @@ require('../app/assets/icon-256x256.png');
 require('../app/assets/icon-384x384.png');
 require('../app/assets/icon-512x512.png');
 
+const TimeoutError = new Error(
+  'React prepare timeout when server side rendering.'
+);
+
+function prepareWithTimeout(app, ms = 15 * 1000) {
```

Comment:
Might as well move ms to a constant here. Also, 15 seconds is probably way too high? Maybe something like 3-5 seconds is more reasonable.

I guess the optimal thing would be if we could figure out why prepare times out, and somehow have it bubble up the error instead?

Reply:
+1 on the constant thing.

Totally agree about the `3-5sec` too, but we have had some discussions regarding timeouts - and `fetchJSON` currently have a `15sec` timeout...

The latency between the server-side renderer and the backend is more or less equal to zero, and that (up to 15sec) slow queries shouln't be run on the server.

Reply:
Will fix

Reply:
And yes, we should do some debugging. But when chaining 2-3-4 actions, with 15sec timeouts - the browser will timeout, and the user will be unable to open the page at all...